code/controllers/subsystem/timer.dm

```diff
@@ -252,6 +260,12 @@ SUBSYSTEM_DEF(timer)
 	// Add all timed events from the secondary queue as well
 	alltimers += second_queue
 
+	for (var/datum/timedevent/t as anything in alltimers)
```

Comment:
Also var name like "t" are not allowed, call it timedevent or smth

Reply:
> I wonder if these changes must be also implemented in other buckets system, like autofire or runechat?

Why runechat isn't based on SStimer on the first place?